lib/Installation/Partitioner/LibstorageNG/v4_3/PartitionFormatMountOptionsPage.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Handles formating and mounting options adding partition
+# using Expert Partitioner.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::Partitioner::LibstorageNG::v4_3::PartitionFormatMountOptionsPage;
+use strict;
+use warnings;
+use parent 'Installation::Partitioner::LibstorageNG::v4_3::FormatMountOptionsPage';
+
+sub init {
+    my ($self) = shift;
+    $self->SUPER::init();
+    $self->{partition_id} = $self->{app}->combobox({id => '"Y2Partitioner::Widgets::PartitionIdComboBox"'});
+    return $self;
+}
+
+sub select_partition_id {
+    my ($self, $partition_id) = @_;
+    return $self unless $partition_id;
```

Comment:
I would rather suggest not to call method if id is not passed instead of doing nothing here and do validation on the caller side.

Reply:
I found this the better option, so we don't have those ugly conditions on the controller, or different methods depending on each partitioning we would select (which would be a mess for looping over partition in the test if for each one we need a different method). Saying on the framework 'fill in all of this depending on the test data' it was the most convincing option for me. The whole PR is based on that, so let's rethink it together if you strong disagree with this.